WifiART/src/structs/LciState.java

```diff
@@ -0,0 +1,65 @@
+package structs;
+
+public class LciState {
```

Comment:
Consider using a Builder pattern. Esp, what are the required / mandatory fields?